src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1254,11 +1213,14 @@ namespace WaterToAirHeatPumpSimple {
 								MixHumRat = OutAirFrac * FinalSysSizing( CurSysNum ).PrecoolHumRat + ( 1.0 - OutAirFrac ) * FinalSysSizing( CurSysNum ).RetHumRatAtCoolPeak;
 							}
 						}
+						// supply air condition is capped with that of mixed air to avoid SHR > 1.0
+						SupTemp = min( MixTemp, SupTemp );
+						SupHumRat = min( MixHumRat, SupHumRat );
 						OutTemp = FinalSysSizing( CurSysNum ).OutTempAtCoolPeak;
 						rhoair = PsyRhoAirFnPbTdbW( StdBaroPress, MixTemp, MixHumRat, RoutineName );
 						MixEnth = PsyHFnTdbW( MixTemp, MixHumRat );
 						MixWetBulb = PsyTwbFnTdbWPb( MixTemp, MixHumRat, StdBaroPress, RoutineName );
-						SupEnth = PsyHFnTdbW( SupTemp, SupHumRat );
+						SupEnth = PsyHFnTdbW( SupTemp, MixHumRat );
```

Comment:
if the performance curves modifier curve values may change the sensible to exceed the total capacity even slightly, then the following statements were added to capture it. See line # 1306.

if ( RatedCapCoolTotalAutoSized && RatedCapCoolSensAutoSized ) {
	if ( RatedCapCoolSensDes > RatedCapCoolTotalDes ) {
	     RatedCapCoolTotalDes = RatedCapCoolSensDes;
	}
}